lms/services/blackboard_api/_basic.py

```diff
@@ -53,11 +53,13 @@ def __init__(
         redirect_uri,
         http_service,
         oauth_http_service,
+        refresh_enabled,
     ):  # pylint:disable=too-many-arguments
         self.blackboard_host = blackboard_host
         self.client_id = client_id
         self.client_secret = client_secret
         self.redirect_uri = redirect_uri
+        self.refresh_enabled = refresh_enabled
```

Comment:
I made this public to make it easy to test. It's not important because the feature flag will soon be removed anyway